components/eam/src/chemistry/modal_aero/aero_model.F90

```diff
@@ -2395,57 +2395,60 @@ subroutine aero_model_gasaerexch( loffset, ncol, lchnk, delt, &
    !------------------------------------------------------
 
     if (mam_amicphys_optaa <= 0) then
-    ! do gas-aerosol exchange, nucleation, and coagulation using old routines
 
-       ! do gas-aerosol exchange (h2so4, msa, nh3 condensation)
-       if (ndx_h2so4 > 0) then
-          del_h2so4_aeruptk(1:ncol,:) = vmr(1:ncol,:,ndx_h2so4)
-       else
-          del_h2so4_aeruptk(:,:) = 0.0_r8
-       endif
+       call endrun("mam_amicphys_optaa <= 0 is nolonger supported")
```

Comment:
We should eventually remove the `mam_amicphys_optaa` option as it is >0 for MAM4. We can remove this "if" condition here.

Reply:
If block and old codes removed in the next commit.